sdk/hdinsight/Microsoft.Azure.HDInsight.Job/AzSdk.RP.props

```diff
@@ -1,7 +1,7 @@
-<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <!--This file and it's contents are updated at build time moving or editing might result in build failure. Take due deligence while editing this file-->
-  <PropertyGroup>
-    <AzureApiTag>HDInsightJobManagementClient_2018-11-01-preview;</AzureApiTag>
-    <PackageTags>$(PackageTags);$(CommonTags);$(AzureApiTag);</PackageTags>
-  </PropertyGroup>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
I think we should just remove this props file and move it into the project file.  I don't see any why this should be separate. 

@shahabhijeet the comment in this file seems to indicate that there is something that mutates this file are you aware of anything that does that?